pages/view/_iscnId/index.vue

```diff
@@ -611,8 +620,10 @@ export default class ViewIscnIdPage extends Vue {
     arg0: string
   ) => Promise<{ records: ISCNRecordWithID[] }>
 
+  @walletModule.Getter('getWalletAddress') currentAddress!: string
+
   get isShowMintButton() {
-    return !this.isPreminted
+    return !this.isPreminted && this.iscnOwner === this.currentAddress
```

Comment:
`isPreminted` is set by calling `api.like.co/likernft/mint`, which doesn't work for nft book
Please also update the condition of setting `isPreminted`, maybe just check if this iscn id has minted any NFT class on chain level, instead of checking api.like.co

Reply:
updated [caa0c09](https://github.com/likecoin/app-like-co/pull/423/commits/caa0c09626c6c9c1c1916e9a3a53f2744fbd2b48)